apps/meteor/ee/client/views/admin/users/UserPageHeaderContentWithSeatsCap.tsx

```diff
@@ -68,7 +50,7 @@ const UserPageHeaderContentWithSeatsCap = ({ activeUsers, maxActiveUsers }: User
 					{t('New_user')}
 				</Button>
 				{isCreateUserDisabled && (
-					<Button is='a' href={manageSubscriptionUrl} target='_blank' rel='noopener noreferrer' primary>
+					<Button mis={8} is='a' href={manageSubscriptionUrl} target='_blank' rel='noopener noreferrer' primary>
```

Comment:
When a button is actually an external link, prefer:
- adding an `onClick` to the button with `useExternalLink` hook rather than `Button is='a' (...)`
- add `role='link` for semantics

Doing that, you won't need to manually add the margin - it will receive from `ButtonGroup` correctly.

Reply:
Thanks for your review! It's fixed now